packages/errors/src/context.ts

```diff
@@ -450,6 +452,11 @@ export type SolanaErrorContext = DefaultUnspecifiedErrorContextToUndefined<
         [SOLANA_ERROR__JSON_RPC__SERVER_ERROR_BLOCK_STATUS_NOT_AVAILABLE_YET]: {
             __serverMessage: string;
         };
+        [SOLANA_ERROR__JSON_RPC__SERVER_ERROR_EPOCH_REWARDS_PERIOD_ACTIVE]: {
+            currentBlockHeight: bigint;
+            rewardsCompleteBlockHeight: bigint;
+            slot: bigint;
```

Comment:
This doesn't actually exist today, but let's pretend that it does. Upon deployment (of the Agave RPC) containing https://github.com/anza-xyz/agave/pull/6962, this will just start to work.

Same with the one below.